packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -73,10 +75,16 @@ export default styled(SplashHandler)`
   flex-direction: column;
   height: 100%;
 
-  > *:not(.splash):not(.header) {
+  > *:not(.splash):not(.header):not(${ScrollWrapper}):not(${Steps}) {
     padding-left: 16px;
     padding-right: 16px;
   }
+  
+  > *:not(.splash):not(.header)${ScrollWrapper} {
```

Comment:
I can't think of a better way to achieve the goal without redesigning the entire skeleton of the app than referencing the children here.